### PR TITLE
Feat: Added Team Members

### PR DESCRIPTION
**Pull request by vinzvinci**
## Related Issue

- Information about the related issue

Closes: #  N/A

### Describe the changes you've made

I created team sections using Tailwind CSS

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

image

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: src/pages/team.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    linkedinUrl: 'https://www.linkedin.com/in/ritvikshukla/',
+  },
+  {
+    name: 'Maxime Kubik',
+    role: 'WebX Core',
+    imageUrl:
+      'https://github.com/mkubdev.png',
+    githubUrl: 'https://github.com/mkubdev',
+    twitterUrl: 'https://twitter.com/mkubdev',
+    linkedinUrl: 'https://www.linkedin.com/in/maximekbk/',
```
Comment (Panquesito7) [inline src/pages/team.js:38]: Question: shouldn't we add the other maintainers as well, or only WebX Core counts? 🤔

File: src/pages/team.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                  </a>
+                </li>
+              </ul>
+            </li>
+          ))}
+        </ul>
+      </div>
+    </div>
+  )
+}
```
Comment (Panquesito7) [inline src/pages/team.js:97]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

File: src/pages/team.js (hunk 1 above)
Comment (vinzvinci) [reply src/pages/team.js:38]: I'm thinking to add them below the WebX Core later using grid with round images.

**Review by shuklaritvik06 — APPROVED** [review]
LGTM

File: src/pages/team.js (hunk 1 above)
Comment (shuklaritvik06) [reply src/pages/team.js:38]: > Question: shouldn't we add the other maintainers as well, or only WebX Core counts? thinking

For us everyone matters equally David! Any Org is by its members then its founders :) I agree with  vinci's thoughts

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🚀

**Comment by vinzvinci** [comment]
Added 
image

**Comment by shuklaritvik06** [comment]
> 
You rock!

**Review by mkubdev — APPROVED** [review]
LGTM @vinzvinci ⚡

**Pull request merged**